sources/kernel-cachyos-bore/kernel-cachyos-lto.spec

```diff
@@ -22,25 +22,16 @@
 %define asmarch x86
 %endif
 
+# define git branch to make testing easier without merging to master branch
+%define _git_branch upgrade-nvidia-f41
```

Comment:
We don't really need since we can just push `master` to the PR and change the branch locally for testing.

Reply:
what do you mean by pushing master to the PR?

Reply:
when it gets merged i will change the value to master

Reply:
We can just keep the URL `https://raw.githubusercontent.com/CachyOS/copr-linux-cachyos/master/sources/kernel-patches/nvidia` but change it locally to `https://raw.githubusercontent.com/CachyOS/copr-linux-cachyos/some-branchsources/kernel-patches/nvidia` to upload to the dev copr.

Reply:
Well on second thought, this seems fine